### DISCUSSION TITLE
DBeaver Community Edition Cannot Connect to DuckDB

### DISCUSSION THREAD

**Discussion by pavlokurochka** [dbeaver/dbeaver#34232] [Q&A] [answered]
### Description

error: Serialization Error: Failed to deserialize: field id mismatch, expected: 201, got: 103

### DBeaver Version

Community  24.0.5.202405191503

### Operating System

Win 11

### Database and driver

duckdb v0.10.3 70fd6a8a24
duckdb_jdbc-0.9.2.jar
[dbeaver-diagnostic-info-1717206723769.zip](https://github.com/user-attachments/files/15520085/dbeaver-diagnostic-info-1717206723769.zip)

### Steps to reproduce

Steps to reproduce
Launch the "Connect to a database" dialogue
Select DuckDB
Select DuckDB db file
Press "Test Connection..."

**Comment by ShadelessFox** [comment, marked as answer]
Hello @pavlokurochka,

It seems that you have quite an outdated driver (the latest version is 0.10.3), try upgrading it.

![image](https://github.com/dbeaver/dbeaver/assets/35821147/69719c17-8f96-445d-ac21-dbd89cd40eb1)

Please let me know if it helps.

**Comment by pavlokurochka** [comment]
Upgrading the diver this way worked. Тhanks a lot!
